tests/unit/src/test/scala/tests/JdkSourcesSuite.scala

```diff
@@ -1,9 +1,26 @@
 package tests
 
 import scala.meta.internal.metals.JdkSources
+import scala.meta.internal.mtags.OnDemandSymbolIndex
+import scala.meta.internal.mtags.Symbol
 
 class JdkSourcesSuite extends BaseSuite {
   test("src.zip") {
     JdkSources.getOrThrow()
   }
+
+  test("index-src.zip") {
```

Comment:
index-modules-src.zip

Reply:
it's not neccessarily modules, we test it also for JDK 8, which doesn't have modules.